client/app/components/text-field/component.js

```diff
@@ -5,6 +5,10 @@ export default Ember.Component.extend({
 
     description: "Enter a title for the preprint.",
 
+    didReceiveAttrs: function() {
+        this.set('description', this.attrs.description);
```

Comment:
I don't think this is necessary, if there is a description passed in as attribute it will override the default.